lib/commonBindings.js

```diff
@@ -133,6 +133,15 @@ function extractAttributes(device, current, payloadType) {
                                     ent.value = entity[k].object;
                                 }
                             }
+                            // Add other stuff as metadata
+                            for (let key in entity[k]) {
+                                if (!['type', 'value', 'object'].includes(key.toLowerCase())) {
```

Comment:
This comes from NGSIv2 allowed fields for attributes? In this case, note `object` is not one of these (only `value`, `type` and `metadata` are allowed, the later one doesn't apply here).

Reply:
object in ngsild is transformed in to value for ngsiv2, but in this point we are consulting ngsild entity, which has object key.

Reply:
So you mean that in NGSI-LD both `value` and `object` are used for attribute value as shown in the following examples?

```
    "status": {
        "type": "Property",
        "value": "free",
        "observedAt": "2018-09-21T12:00:00Z"
    },
```

```
    "refParkingSite": {
        "type": "Relationship",
        "object": "urn:ngsi-ld:ParkingSite:santander:daoiz_velarde_1_5"
    },
```

Reply:
Yes, these keys are already handled and should not added to metadata as other stuff.

Reply:
NTC